src/dmd/cli.d

```diff
@@ -194,90 +295,206 @@ struct Usage
             "ignore unsupported pragmas"
         ),
         Option("inline",
-            "do function inlining"
+            "do function inlining",
+            `inline expand functions at the discretion of the compiler.
```

Comment:
The "inline expand" looks out of place. I would just say "Inline functions"

Reply:
I agree.  Sorry, I didn't comment about  that earlier.  Care to make the change to "Inline functions..." before we merge?